underdocs-renderer/src/main/kotlin/underdocs/renderer/parser/section/ErrorHandlingParser.kt

```diff
@@ -0,0 +1,55 @@
+package underdocs.renderer.parser.section
+
+import com.vladsch.flexmark.ast.BulletList
+import com.vladsch.flexmark.ast.BulletListItem
+import com.vladsch.flexmark.util.ast.Document
+import com.vladsch.flexmark.util.ast.Node
+import underdocs.renderer.representation.documentation.ErrorHandling
+import underdocs.renderer.representation.documentation.ErrorHandlingItem
+
+class ErrorHandlingParser : AttemptingSectionParser<ErrorHandling>() {
+    override fun canParse(document: Document) =
+        document.children.any { isSectionHeadingWithTitle(it, "Error Handling") }
+
+    override fun parse(document: Document): ErrorHandling {
+
```

Comment:
Just a note. Will be fixed by adopting a linter/formatter.

Reply:
Whoops, sorry about that!

Reply:
No probs, only noted it so that nobody would think I'm being lazy not mentioning it 😄